src/java/com/phenix/pct/PCTRun.java

```diff
@@ -890,6 +925,14 @@ protected void cleanup() {
                     }
                 }
             }
+            if (runAttributes.getXCodeInit()) {
+                if (!new File(xcodeDir, initProc.getName()).delete()) {
+                    log("Unable to delete xcode'd procedure"); //$NON-NLS-1$
+                }
+                if (!xcodeDir.delete()) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use "java.nio.Files#delete" here for better messages on error conditions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS4042)